delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -39,21 +42,41 @@ def __init__(self, **kwargs):
         self.rest_port = rest_access.get('port')
         self.rest_username = rest_access.get('username')
         self.rest_password = rest_access.get('password')
+        super(RestClient, self).__init__()
         # Lists of addresses to try, for authorization
         self.san_address = [
-            'https://' + self.rest_host + ':' + self.rest_port +
+            'https://' + self.rest_host + ':' + str(self.rest_port) +
             '/deviceManager/rest/']
         self.session = None
         self.url = None
         self.device_id = None
+        self.verify = None
+        urllib3.disable_warnings(InsecureRequestWarning)
+        self.reset_connection(**kwargs)
+
+    def reset_connection(self, **kwargs):
+        self.verify = kwargs.get('verify', False)
+        try:
+            self.login()
+        except Exception as ex:
+            msg = "Failed to login to OceanStor: {}".format(ex)
+            LOG.error(msg)
+            raise exception.InvalidCredential(msg)
 
     def init_http_head(self):
         self.url = None
         self.session = requests.Session()
         self.session.headers.update({
             "Connection": "keep-alive",
             "Content-Type": "application/json"})
-        self.session.verify = False
+        if not self.verify:
+            self.session.verify = False
```

Comment:
When verify is disabled, I think we also need to remount the adapter

Reply:
When verify is disabled, it works without remount the adapter